04 Event Emitters/assets/js/app/components/TableComponent.js

```diff
@@ -0,0 +1,59 @@
+(function ComponentInitializator($, React, App) {
+  'use strict';
+
```

Comment:
The scope of this autoinvoked function, is just this Ract element, the name ComponentInitializator, in my humble opinion, is too generic, I will use instead, tableComponentInitialize. And why, do you use Pascal case for the name?